ls1intum/Artemis
# `Programming exercises`: Fix URL to SSH key settings in the Gitlab setup

### Checklist
#### General



### Motivation and Context
Gitlab changed the URL to access the user profile settings.

### Description
Updates the config file to mention the new URL.

As this does not actually changes the configuration value due to the default being Bamboo setups, it may be worth informing admins in the changelog that a manual configuration change may be necessary.

### Steps for Testing
Prerequisites:
- A local installation with Gitlab.

1. You probably have got the old setting in your config file. When clicking the ‘Clone Repository’ button and selecting SSH, the dialogue contains a link to the Gitlab settings where you can add a key. This should lead to a 404.
2. Update the `ssh-keys-url-path` as per the changes of this PR in your configuration files and restart Artemis.
3. The same link in the pop-up should now work. 

### Review Progress





#### Code Review
#### Manual Tests
Probably optional.

### Test Coverage
unchanged

### Screenshots
n/a
